plugins/woocommerce-admin/client/products/use-product-helper.ts

```diff
@@ -159,26 +157,24 @@ export function useProductHelper() {
 				status,
 			} ).then(
 				( updatedProduct ) => {
+					const noticeContent =
+						product.status === 'draft' &&
+						updatedProduct.status === 'publish'
+							? __(
+									'Product published. View in store.',
+									'woocommerce'
+							  )
+							: __(
+									'Product successfully updated.',
+									'woocommerce'
+							  );
 					if ( ! skipNotice ) {
-						createNotice(
-							'success',
-							product.status === 'draft' &&
-								updatedProduct.status === 'publish'
-								? __(
-										'🎉 Product published. View in store.',
-										'woocommerce'
-								  )
-								: __(
-										'🎉 Product successfully updated.',
-										'woocommerce'
-								  ),
-							{
-								actions: getNoticePreviewActions(
-									updatedProduct.status,
-									updatedProduct.permalink
-								),
-							}
-						);
+						createNotice( 'success', `🎉‎ ${ noticeContent }`, {
```

Comment:
Same as previous comment

Reply:
@mdperez86 there is [a related issue ](https://github.com/woocommerce/woocommerce/issues/35716)that removes the duplicated copy `Product published. View in store.',`. Could we handle it here? = Remove "`View in store.',"`.

Reply:
Sure, so we can close [this one](https://github.com/woocommerce/woocommerce/issues/35716)